test/runtests.jl

```diff
@@ -3,6 +3,7 @@ using Pkg
 using QuantumToolbox
 using QuantumToolbox: position, momentum
 using Random
+using SciMLOperators
```

Comment:
If we don't need to use `MatrixOperator` above, then I think we can remove this.

Reply:
We need it to increase the coverage